src-tauri/src/pipeline/merge_exposures.rs

```diff
@@ -43,15 +39,21 @@ pub fn merge_exposures(
     command.arg("-f");
     command.arg("-g");
 
+    // Add input LDR images as args
```

Comment:
The argument order seems to have gotten switched up - I think the LDR image args should be placed back where it was, right below where the command is created and before any other args.

Reply:
I believe this is still the incorrect order of arguments at the moment